css/component_styles/alert.css

```diff
@@ -0,0 +1,64 @@
+/* -------- alert css -------- */
+:root {
+    --color-success: #38A169;
+    --color-warning: #DD6B20;
+    --color-error: #E53E3E;
+    --color-info: #4C95D6;
+    --color-success-bg: #C6F6D5;
+    --color-warning-bg: #FEEBC8;
+    --color-error-bg: #FED7D7;
+    --color-info-bg: #BEE3F8;
+}
+
+.alert {
+    font-family: Arial, Helvetica, sans-serif;
+    width: 30rem;
+    margin: 5px auto;
+    padding: 1rem;
+    background-color: var(--bg-color);
+    border-radius: 2px;
+    border: solid 1px transparent;
+}
+
+/* simple alert */
+.primary {
```

Comment:
In class `.primary` you are just adding background-color. you can make this a common class for the entire project by adding it in your main css file. This way it allows you to define one and use many times

Reply:
ok thanks i will do this 👍